src/torchcodec/_frame.py

```diff
@@ -114,3 +114,28 @@ def __len__(self):
 
     def __repr__(self):
         return _frame_repr(self)
+
+
```

Comment:
Below: the audio decoding API returns the new `AudioSamples` class rather than a pure Tensor. I think it has the following benefits:

- users can keep track of the sample_rate within that struct without having to handle it separately
- for some edge-cases, like with our mp3 test asset, the stream's beginning isn't 0. So we also returns `pts_seconds`, which may not always be equal to `start_seconds`.

Reply:
I think this makes sense, and it mirrors the video API. In the video API, `__getitem__()` returns just the tensor, but the named methods return `Frame` or `FrameBatch`. I think we should probably do that for audio as well.